ui/apps/platform/cypress/integration/compliance-enhanced/ComplianceEnhanced.helpers.js

```diff
@@ -1,14 +1,56 @@
+import { visitFromLeftNavExpandable } from '../../helpers/nav';
 import { visit } from '../../helpers/visit';
 
 // path
 
 export const basePath = '/main/compliance-enhanced';
+export const statusDashboardPath = `${basePath}/status`;
+export const scanConfigsPath = `${basePath}/scan-configs`;
 
-// visit
+// visit helpers
+export const scanConfigsAlias = 'configurations';
+
+const routeMatcherMapToComplianceEnhancedDashboard = null;
+
+const routeMatcherMapToComplianceEnhancedScanConfigs = {
+    [scanConfigsAlias]: {
+        method: 'GET',
+        url: '/v2/compliance/scan/configurations*',
+    },
+};
+
+export function visitComplianceEnhancedFromLeftNav() {
+    visitFromLeftNavExpandable(
+        'Compliance (2.0)',
+        'Compliance Status',
+        routeMatcherMapToComplianceEnhancedDashboard
+    );
+
+    cy.location('pathname').should('eq', statusDashboardPath);
+}
 
 export function visitComplianceEnhancedDashboard() {
     // TODO: (vjw, 1 Nov 2023) add routes matchers to this function, after API is available for Status
-    visit(`${basePath}/status`);
+    visit(basePath, routeMatcherMapToComplianceEnhancedDashboard);
 
+    cy.location('pathname').should('eq', statusDashboardPath);
```

Comment:
The intent here is to test the redirect on every test?

Reply:
Good point, I should move the redirect test to an individual test.